tests/translator/input/error_api_with_invalid_path_object.yaml

```diff
@@ -0,0 +1,47 @@
+Globals:
+  Api:
+    Name: "some api"
+    Variables:
+      SomeVar: Value
+    Auth:
+      DefaultAuthorizer: MyCognitoAuth
+      Authorizers:
+        MyCognitoAuth:
+          UserPoolArn: !GetAtt MyUserPool.Arn
+
+Resources:
+  ImplicitApiFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://sam-demo-bucket/member_portal.zip
+      Handler: index.gethtml
+      Runtime: nodejs12.x
+
+  ExplicitApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: SomeStage
+      DefinitionBody:
+        swagger: 2.0
+        paths:
+          "/a":
```

Comment:
the httpMethod and paths dont line up with the test itself.

Reply:
I swear this worked locally. Good catch.